SYCL/InvokeSimd/popcnt.cpp

```diff
@@ -0,0 +1,185 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// Check that full compilation works:
+// RUN: %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out
+// RUN: env IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+//
+// VISALTO enable run
+// RUN: env IGC_VISALTO=63 IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+
+// Tests invoke_simd support in the compiler/headers
+
+/* Test case purpose:
+ * -----------------
+ * This invoke_simd program has been modeled after
+ * https://gitlab.devtools.intel.com/dhochman/sginvokesimd_tests/-/blob/master/popcnt/popcnt.cpp
+ * which is not a working program, rather some pre-existing test case
+ * that seems to be testing simd_mask functionality and the popcount()
```

Comment:
What does it mean 'seems'? popcnt is orthogonal to invoke_simd. What aspect of invoke_simd implementation this test checks, which is not checked by other tests?